gobblin-metrics/src/main/java/gobblin/metrics/context/filter/ContextFilterFactory.java

```diff
@@ -30,30 +29,30 @@
 
   /**
    * Modify the configuration to set the {@link ContextFilter} class.
-   * @param config Input {@link Config}.
+   *
+   * @param props the {@link Properties} too add the {@link ContextFilter} specification to
    * @param klazz Class of desired {@link ContextFilter}.
-   * @return Modified {@link Config}.
    */
-  public static Config setContextFilterClass(Config config, Class<? extends ContextFilter> klazz) {
-    return config.withValue(CONTEXT_FILTER_CLASS, ConfigValueFactory.fromAnyRef(klazz.getCanonicalName()));
+  public static void setContextFilterClass(Properties props, Class<? extends ContextFilter> klazz) {
```

Comment:
I would prefer to leave this class with Config instead of Properties. This class was created for use with metrics reporting config files which need features from Config.